perf/src/bin/perf_client.rs

```diff
@@ -48,6 +48,9 @@ struct Opt {
     /// Receive buffer size in bytes
     #[structopt(long, default_value = "2097152")]
     recv_buffer_size: usize,
+    /// Specify the local ip
+    #[structopt(long)]
+    local_ip: Option<IpAddr>,
```

Comment:
```suggestion
    /// Specify the local address
    #[structopt(long)]
    local_addr: Option<SocketAddr>,
```

Reply:
I'll update it to use a socketaddr soon :)